server/lib/facebook/facebook_feed_stream.js

```diff
@@ -2,30 +2,23 @@
  * Module streas facebook feed
  **/
 
-import { PassThrough } from 'stream';
+import AbstractFeedStream from './../abstract_feed_stream';
```

Comment:
You could remove the leading dot for better readability. Maybe it's also worth thinking of setting a NODE_PATH to avoid using relative paths?